index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio: setup and mobile first</title>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="logo-placeholder">
+          <h1>Otmane Ech</h1>
+        </div>
+        <div class="icon">
+          <img src="./assets/Enabled.png" alt="Enabled" />
+        </div>
+      </header>
+      <section class="headlines">
+        <h1 class="primary-text">
+            Hey There. I’m Otmane.</h1>
+        <h2 class="supporting-text">
+            I'm a Software Developer
+        </h2>
+        <p class="description">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+        </p>
+        <ul class="icons-container">
+          <li class="icon"><img src="./assets/github.png"></li>
+          <li class="icon"><img src="./assets/In.png" alt=""></li>
+          <li class="icon"><img src="./assets/fin.png" alt=""></li>
+          <li class="icon"><img src="./assets/twitter.png" alt=""></li>
+          <li class="icon"><img src="./assets/m.png" alt=""></li>
+        </ul>
```

Comment:
- [ ] Kindly nest all your social icons in an anchor tag with `href` attribute pointing to the social handle. THis is better user experience.
```html
<ul>
  <li>
    <a><img /></a>
  </li>
...
```

Reply:
Okay, Thank you for showing the exact weak points